chemie/customprofile/templates/customprofile/yearbook.html

```diff
@@ -122,6 +122,13 @@ <h4 class="grey-text darken-4" style="margin-top: 70px;">{{ profile }}</h4>
                 <p>E-post: <a href="mailto:{{ profile.user.email }}">{{ profile.user.email }}</a></p>
                 <p>Mobilnummer: {{ profile.phone_number }}</p>
                 <p>Samlivsstatus: {{ profile.get_nice_relationship_status }}</p>
+                <p>Digidaljer:</p>
```

Comment:
Perhaps remove this and just leave the medal pictures? What do you think, @yoonsiko?